testing-overhead-jmh/src/jmh/java/io/opentelemetry/javaagent/benchmark/servlet/ServletBenchmark.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.benchmark.servlet;
+
+import io.opentelemetry.javaagent.benchmark.servlet.app.HelloWorldApplication;
+import io.opentelemetry.testing.internal.armeria.client.WebClient;
+import java.util.concurrent.TimeUnit;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.TearDown;
+
+@BenchmarkMode(Mode.SampleTime)
+@OutputTimeUnit(TimeUnit.MICROSECONDS)
+@State(Scope.Thread)
+public class ServletBenchmark {
+
+  static {
+    HelloWorldApplication.main();
```

Comment:
Just wondering: why do we initialize the app in `static {}` and not `@Setup`?

Reply:
added comment to explain

```
    // using static initializer instead of @Setup since only want to initialize the app under test
    // once regardless of @State and @Threads
```